contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,294 @@
+  <ComparisonRow column1={true} column2={true} feature="Bootstrapping" description="Flags available on frontend application load" />
+  <ComparisonRow column1={true} column2={false} feature="Early access management" description="Manage betas, test features" />
+</ComparisonTable>
+
+### Product analytics
+
+Both PostHog and Statsig offer the functionality you expect from product analytics tools, such as dashboards, graphs, and funnels. However:
+
+- **PostHog** provides deep insights into how users are interacting with your product. It includes features such as lifecycle, stickiness, correlation, and retention analysis.
+
+- **Statsig** does provide some additional insights, such as retention analysis, but its main focus is to use product analytics to set up metrics to run A/B tests on. 
+
+<ComparisonTable column1="PostHog" column2="Statsig">
+  <ComparisonRow column1={true} column2={false} feature="Autocapture" description="Capture events without manual logging" />
+  <ComparisonRow column1={true} column2={true} feature="Dashboards" description="Combine insights into shareable dashboards" />
+  <ComparisonRow column1={true} column2={true} feature="Graphs and trends" description="Build custom insights and visualizations" />
+  <ComparisonRow column1={true} column2={true} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
+  <ComparisonRow column1={true} column2={true} feature="Group analytics" description="Track metrics at a company level" />
+  <ComparisonRow column1={true} column2={true} feature="Funnels" description="Track users through a sequence of events" />
+  <ComparisonRow column1={true} column2={true} feature="Retention analysis" description="Visualize which users stay, for how long" />
+  <ComparisonRow column1={true} column2={false} feature="User paths" description="Track user flows and where they drop-off" />
+  <ComparisonRow column1={true} column2={false} feature="Correlation analysis" description="Suggested events and properties that lead to success or failure" />
+  <ComparisonRow column1={true} column2={false} feature="Lifecycle analysis" description="Understand who is dormant, churning, and thriving" />
+  <ComparisonRow column1={true} column2={false} feature="Stickiness insights" description="See how many times users perform an event in a period of time." />
+  <ComparisonRow column1={true} column2={false} feature="Formulas" description="Use custom formulas to calculate unique insights" />
+  <ComparisonRow column1={true} column2={false} feature="Query editor" description="Write your own queries in SQL" />
+</ComparisonTable>
+
+### Pricing
+
+Both PostHog and Statsig offer simple, transparent pricing.
+
+**Statsig** has a free plan for up to 1 million *metered events* (a metered event is any feature flag, A/B testing, or logging event). Thereafter, it's $150 per month for up to 5 million metered events, and then $50 per every 1 million events. 
```

Comment:
warehouse native also has its own plan.

Reply:
I think it's fine to not mention it. This blog doesn't focus too much on warehouse comparison, and I suspect that companies want warehouse native will expect to pay extra for it.